shell/common/shell.h

```diff
@@ -689,6 +689,10 @@ class Shell final : public PlatformView::Delegate,
   // |Rasterizer::Delegate|
   fml::TimePoint GetLatestFrameTargetTime() const override;
 
+  // |Rasterizer::Delegate|
+  bool ShouldDiscardLayerTree(int64_t view_id,
+                              flutter::LayerTree& tree) override;
```

Comment:
Can this be `const`?

```suggestion
                              const flutter::LayerTree& tree) override;
```